### DISCUSSION TITLE
Option for closed Object Browser?

### DISCUSSION THREAD

**Discussion by patrickgardella** [R-nvim/R.nvim#192] [Q&A] [answered]
I am trying to find a way to have the Object Browser open the .GlobalEnv objects with the elements hidden by default.  I can go through and hit <enter> or double click each object, which closes them, but it would be easier to read and navigate if the elements were not all listed.

I'm working with NOAA weather data, and there are a _lot_ of elements.

Is there a configuration option? I couldn't find anything in the wiki, documentation pages, a Google search, or attempting to read the source code (not a Lua developer yet).

Thanks!

**Comment by jalvesaq** [comment, marked as answer]
Please, see `objbr_opendf` in section _6.7. Object Browser and completion options_ of `R.nvim` documentation:

```vim
:help objbr
```

**Reply by patrickgardella** [reply]
Perfect, thanks!  (Yes, that was pretty obvious, but somehow I missed it.)

For others, here is how I configured it with lua config files:
```
{
    "R-nvim/R.nvim",
    config = function()
        local rnvim = require("r")
        rnvim.setup({
            objbr_opendf = false,
            objbr_openlist = false,
        })
    end,
    lazy = false,
},
```
